### DISCUSSION TITLE
[DONE][Beats - fleet-server] Spring Clean up Session

### DISCUSSION THREAD

**Discussion by jlind23** [elastic/beats#30213] [Ideas]
# Spring Clean up Session? What is that?
As owners of the Beats repository we should keep it as pleasant as possible for you to work with. First step is to make it more discoverable by removing stuff that does not make sense anymore.

## From the 4th to the 8th of April, Elastic engineers will do a Spring Clean up Session on both Beats and fleet-server repository.

The overall target is to reduce the amount of open issues and open PR. (Obviously if those are still relevant it should stay open)

This github conversation will act as an Advisory board, if there is an item with which you do not know what to do, feel free to ask for help here by starting a new conversation thread.
A solution as the conversation thread is resolved, it can be hidden.

On the first day I will post here our starting point in terms of PR and issues numbers and let see how far we can go.

Having less than 20 PR open at a point in time seems to be manageable and the way to go.

How to proceed?
1. Take care of what belongs to you by using the URLs below:
1.1 https://github.com/elastic/beats/pulls/your_github_id
1.2 https://github.com/elastic/beats/issues/your_github_id

2. Take care of what was assigned to you:
2.1 https://github.com/elastic/beats/pulls/assigned/your_github_id
2.2 https://github.com/elastic/beats/issues/assigned/your_github_id

3. As soon as you work on an issue/PR which is neither assigned or authored by you than you first need to assign yourself. It will avoid having someone else looking at it in parallel.

Update 03/30/2020:
- 1318 Open issues - https://github.com/elastic/beats/issues
- 152 Open PRs - https://github.com/elastic/beats/pulls

Update 04/04/2020:
- 1144 Open issues - https://github.com/elastic/beats/issues
- 89 Open PRs - https://github.com/elastic/beats/pulls

Update 04/05/2020:
- 1062 Open issues - https://github.com/elastic/beats/issues
- 89 Open PRs - https://github.com/elastic/beats/pulls

Update 04/07/2020:
- 1031 Open issues - https://github.com/elastic/beats/issues
- 71 Open PRs - https://github.com/elastic/beats/pulls

Update 04/11/2020:
- 1031 Open issues - https://github.com/elastic/beats/issues
- 59 Open PRs - https://github.com/elastic/beats/pulls

**Comment by jlind23** [comment]
Other task to perform: Issues labeled with "Team:Services" must be reassigned to the appropriate team.

https://github.com/elastic/beats/issues?q=is%3Aopen+is%3Aissue+label%3ATeam%3AServices

Update 03/30/2022 - 202 issues wrongly labeled
Update 03/31/2022 - 0 issues wrongly labeled.

**Reply by jlind23** [reply]
@andresrc all issues were cleaned up and reassigned. If needed you can delete this label.

**Comment by jlind23** [comment]
As agree with @cmacknz this test class should be removed as it has been disabled for more than years.
https://github.com/elastic/beats/blob/8.2/libbeat/tests/system/test_monitoring.py
Can someone please pick it up?

@elastic/elastic-agent-data-plane

**Reply by cmacknz** [reply]
I'll delete it.
